src/lightbox/LightboxActionSheet.js

```diff
@@ -36,7 +36,7 @@ type ButtonType = {|
 const tryToDownloadImage = async ({ src, auth }: DownloadImageType) => {
   const tempUrl = await api.tryGetFileTemporaryUrl(src, auth);
   if (tempUrl === null) {
-    showToast('Please download the image from your browser');
+    showToast('Download failed. Please try again from your browser.');
```

Comment:
Hmm, is "download failed" accurate here? Or is it more like "one form of download wasn't available, so we'll use another"?

I looked at our jsdoc for this API method, and the one it calls, and didn't come out with a clear answer. :-/ It seems this API route isn't documented. But I *think* the story is basically that on a server that doesn't have this newer feature, we'll just always use this path for the download instead of the other path.

Reply:
I kind of think the right thing here may be to not have a toast at all. We're opening the file directly in a browser. The user will see it there, and if they want it further "downloaded" they can find the "download" option in their browser. (… Or if they can't, this toast isn't likely to help them.)